pkg/ebpf/c/common/task.h

```diff
@@ -85,12 +85,7 @@ statfunc struct pid_namespace *get_task_pid_ns(struct task_struct *task)
     }
 
     level = BPF_CORE_READ(pid, level);
-    return BPF_CORE_READ(pid, numbers[level].ns);
-}
-
-statfunc u32 get_task_pid_ns_id(struct task_struct *task)
-{
-    struct pid_namespace *ns = get_task_pid_ns(task);
+    ns = BPF_CORE_READ(pid, numbers[level].ns);
```

Comment:
I think we can keep the changes in this file since they are not directly related to the removed events but a refactor